build/source/engine/systemSolv.f90

```diff
@@ -2729,10 +2326,13 @@ real(dp),intent(out)           :: grad(:)       ! gradient of the function vecto
       aJac(iState,iJac) = aJac(iState,iJac)/fscale(kState)
      end do  ! looping through elements of the band-diagonal matric
     end do  ! looping through state variables
-    !print*, '** analytical Jacobian:'
-    !write(*,'(a4,1x,100(i11,1x))') 'xCol', (iLayer, iLayer=iJac1,iJac2)
-    !do iLayer=kl+1,nBands; write(*,'(i4,1x,100(e11.5,1x))') iLayer, aJac(iLayer,iJac1:iJac2); end do
-    !pause
+
+    ! check
+    if(printFlag)then
+     print*, '** banded analytical Jacobian:'
+     write(*,'(a4,1x,100(i11,1x))') 'xCol', (iLayer, iLayer=iJac1,iJac2)
+     do iLayer=kl+1,nBands; write(*,'(i4,1x,100(e17.10,1x))') iLayer, aJac(iLayer,iJac1:iJac2); end do
```

Comment:
Follow SUMMA code conventions for multiple statements on a single line (i.e. don't unless error handling or case statements).